src/rill.h

```diff
@@ -95,6 +95,8 @@ void rill_pairs_print(const struct rill_pairs *pairs);
 // store
 // -----------------------------------------------------------------------------
 
+enum rill_column_e { rill_column_a = 0, rill_column_b };
```

Comment:
If the numbers matter (aka. used as the index in an array) then I would specify both values to be extra clear and sure.

Reply:
Also what does the `_e` stand for? Why not just call the enum `rill_column` or `rill_col`?

Reply:
enum :scream: I can rename this later

Reply:
So adding a type suffix is only really useful in the case of typedef. Since I don't typedef my structs and unions then you'll always have to specify the `enum` keyword before the name, then there's no ambiguity and the `_e` is a bit redundant.

Reply:
Fixorzed